cosmic-usage-ui/src/main/resources/public/app/main.js

```diff
@@ -36,11 +51,74 @@ const app = {
 
         $(this.monthSelectorComponent).datepicker('setDate', new Date());
         this.renderPrintingHeaders();
+        this.renderDomainTableHeaders();
         this.renderDomainsList();
 
         $(this.generateReportButton).on('click', this.generateReportButtonOnClick);
     },
 
+    sortDomainTableByHeader: function(e) {
+        var currState = $(e.target).data(this.domainTableHeaderDataState);
```

Comment:
I'd give `$(e.target)` a name since it's used twice and would help knowing what it is: `const header = $(e.target)`.